.scripts/checkEverything.ts

```diff
@@ -0,0 +1,67 @@
+import { checkEverything, contains, getArgument, getDefaultLogger, gitDiff, GitDiffResult, gitStatus, GitStatusResult, joinPath, Logger, normalize, resolvePath } from "@ts-common/azure-js-dev-tools";
+import * as path from "path";
+import { getPackageFolderPaths } from "./common";
+
+const logger: Logger = getDefaultLogger();
+const changedFiles: string[] = [];
+
+let headReference: string | undefined = getArgument("head-reference", { environmentVariableName: "headReference" });
+if (!headReference) {
+  const statusResult: GitStatusResult = gitStatus();
+  headReference = statusResult.localBranch!;
+  logger.logInfo(`No head-reference argument specified on command line or in environment variables. Defaulting to "${headReference}".`);
+
+  const modifiedFiles: string[] | undefined = statusResult.modifiedFiles;
```

Comment:
We should probably some day "merge" this and [this code](https://github.com/Azure/azure-sdk-for-js/blob/master/.scripts/git.ts#L126) so we don't have two implementations doing pretty much same thing.

Reply:
I thought about doing that, but then decided to save it for another day.

Reply:
Absolutely, agree.